src/app/products/_components/product-list.tsx

```diff
@@ -0,0 +1,63 @@
+"use client";
+
+import { Pagination } from "@/components/ui/pagination";
+import { ROUTE } from "@/configs/route";
+import { getTotalPages } from "@/lib/pagination";
+import { useProducts } from "@/services/product";
+import { redirect } from "next/navigation";
+import { ReactNode } from "react";
+import { useProductListPageQuery } from "../_hooks/use-product-list-page-query";
+import { ProductListItem } from "./product-list-item";
+
+const ITEMS_PER_PAGE = 6;
+
+export const ProductList = () => {
+  const { page, setPage } = useProductListPageQuery();
+
+  const { data: products } = useProducts({
+    params: {
+      page: page,
+      limit: ITEMS_PER_PAGE,
```

Comment:
요런거 신경써주시면 좋을 것 같아요.
```suggestion
    params: {
      page,
      limit: ITEMS_PER_PAGE,
```